dpdata/lammps/dump.py

```diff
@@ -197,6 +197,46 @@ def load_file(fname: FileType, begin=0, step=1):
                 buff.append(line)
 
 
+def get_spin(lines):
+    blk, head = _get_block(lines, "ATOMS")
+    heads = head.split()
+    """
+    the spin info is stored in c_spin[1], c_spin[2], c_spin[3], c_spin[4] or sp, spx, spy, spz, which is the spin norm and the spin vector
+ITEM: ATOMS id type x y z c_spin[1] c_spin[2] c_spin[3] c_spin[4] c_spin[5] c_spin[6] c_spin[7] c_spin[8] c_spin[9] c_spin[10]
+1 1 0.00141160 5.64868599 0.01005602 1.54706291 0.00000000 0.00000000 1.00000000 -1.40772100 -2.03739417 -1522.64797384 -0.00397809 -0.00190426 -0.00743976
+    """
```

Comment:
no code should be placed before the python doc string. 
please follow the convention https://peps.python.org/pep-0257/ to write doc strings.